Training Update: OMERO suite tutorial with new credential system (#6678)
galaxyproject/training-material: topics/imaging/tutorials/omero-suite/tutorial.md

{: .agenda}

## Before Starting - Secure Your OMERO credentials with Galaxy's Credentials System

Before starting you need to set up your OMERO credential. 

**Galaxy 25.1** offers a comprehensive tool credentials system that makes it easy and secure for both users and tool developers to work with external services. All sensitive information (secrets like API keys, passwords, tokens) is now stored encrypted in Galaxy's vault system.

Setting up the credentials for a specific OMERO tool is easy and allows to save multiple OMERO instance within one tool!

When opening an OMERO tool you will see the following orange windows above the tool parameters. This means that no credentials are set-up at the moment. 

Click on **"Provide Credentials"**

![credential_1.png](../../images/omero-suite/credential_1.png)

> A new windows will open. Click on **OMERO Credentials** and then on **+ Create new group**

Reviewer: ```suggestion
A pop-up window will open. Click on **OMERO Credentials** and then on **+ Create new group**
```

![credential_2.png](../../images/omero-suite/credential_2.png)

Add now a **Group Name** and input your OMERO username and password. Optionally, you can use the **OMERO UUID Session Key** for [increased security](https://omero.readthedocs.io/en/stable/users/cli/sessions.html). Click now **Create Group**.

![credential_3.png](../../images/omero-suite/credential_3.png)

Your group is now created! Click on **Use this group** to select this credentials and then **Save Groups Selection**.

![credential_4.png](../../images/omero-suite/credential_4.png)

The window above the tool parameters will now turn green! This mean that the tool is ready to use.

![credential_5.png](../../images/omero-suite/credential_5.png)

**Really important:** each tool need its own credentials... but do not worry, this needs to be done just one time!

> You can find [Something better here](https://artifact.galaxyproject.org/news/2026-01-12-tool-credentials-system/) more information for the Galaxy credential system.

Reviewer: ```suggestion
You can find [here](https://artifact.galaxyproject.org/news/2026-01-12-tool-credentials-system/) more detailed information on the Galaxy credential system.
```

> <hands-on-title>Upload images into OMERO</hands-on-title>
>

> > 1. {% tool [OMERO Image Import](toolshed.g2.bx.psu.edu/repos/ufz/omero_import/omero_import/5.18.0+galaxy6) %} with the following recommended parameters:

Author: Hi @shiltemann the section  [_Create a new workflow in the workflow editor._](https://github.com/rmassei/training-material/blob/751bc9499befd7e0e1eb4916eee84ce8273c53c8/topics/imaging/tutorials/omero-suite/tutorial.md?plain=1#L306) was already updated with the new tool bumps. 
A .ga file is not provided for this training.
